ironfish-cli/src/ui/table.ts

```diff
@@ -210,6 +216,9 @@ class Table<T extends Record<string, unknown>> {
       }
       this.options.printLine(` ${rowValues.join(' ')}`)
     }
+    if (this.options.limit && this.options.limit !== 0 && rows.length >= this.options.limit) {
```

Comment:
Nit: I wonder if `this.options.limit` can be negative and, if so, if we should change `!== 0` to `<= 0`

Reply:
Wouldn't hurt to change it to this!